understanding/accessible-authentication-minimum.html

```diff
@@ -336,97 +252,69 @@ <h3>Object Recognition</h3>
                </section>
             </section>
             <section id="benefits">
-               <h2>Benefits</h2>
-               <p>People with cognitive issues relating to memory, reading (for example, dyslexia),
-                  numbers (for example, dyscalculia), or perception-processing limitations will be able
-                  to authenticate irrespective of the level of their cognitive abilities.
+               <h2>利点</h2>
+               <p>記憶、読解（例えばディスレクシア）、数字（例えばディスカリキュリア）、又は知覚処理制限に関する認知的問題のある人でも、認知能力のレベルに関係なく認証できる。
                </p>
             </section>
             <section id="examples">
-               <h2>Examples</h2>
-               <p>The examples of this Success Criterion are the same as the <a href="accessible-authentication-enhanced.html#examples">Accessible Authentication (Enhanced)</a> examples.
+               <h2>事例</h2>
+               <p>この達成基準の例は、<a href="accessible-authentication-enhanced.html#examples">アクセシブルな認証 (高度)</a> の例と同じである。
                </p>
                <ul>
                   
-                  <li>A web site uses a properly marked up username (or email) and password fields as the
-                     login authentication (meeting <a href="identify-input-purpose">Success Criterion 1.3.5 Input Purpose</a> and <a href="name-role-value">Success Criterion 4.1.2: Name, Role, Value</a>). The user's browser or integrated third-party password manager extension can identify
-                     the purpose of the inputs and automatically fill in the username and password.
+                  <li>ウェブサイトは、適切にマークアップされたユーザ名 (又は電子メール) 及びパスワードのフィールドをログイン認証として使用する (<a href="identify-input-purpose">達成基準 1.3.5 入力目的の特定</a>及び<a href="name-role-value">達成基準 4.1.2: 名前 (name)・役割 (role)・値 (value)</a> を満たしている)。利用者のブラウザ又は統合されたサードパーティのパスワードマネージャー拡張機能は、入力の目的を特定し、ユーザ名及びパスワードを自動的に入力できる。
                   </li>
                   
-                  <li>A web site does not block paste functionality. The user is able to use a third-party
-                     password manager to store credentials, copy them, and paste them directly into a login
-                     form.
+                  <li>ウェブサイトは貼り付け機能をブロックしていない。利用者は、サードパーティのパスワードマネージャーを使用して資格情報を保存し、コピーしてログインフォームに直接貼り付けることができる。
                   </li>
                   
-                  <li>A web site uses WebAuthn so the user can authenticate with their device instead of
-                     username/password. The user's device could use any available modality. Common methods
-                     on laptops and phones are facial-scan, fingerprint, and PIN (Personal Identification
-                     Number). The web site is not enforcing any particular use; it is assumed a user will
-                     set up a method that suits them.
+                  <li>ウェブサイトは WebAuthn を使用するため、利用者はユーザ名/パスワードの代わりにデバイスで認証できる。利用者のデバイスは、利用可能な任意のモダリティを使用できる。ノートパソコン及び電話での一般的な方法は、顔スキャン、指紋、PIN (個人識別番号) である。このウェブサイトは、特定の使用を強制するものではない。利用者が自分に合った方法を設定することを想定している。
```

Comment:
ここで言う phones は、つまりスマホだと思います。
```suggestion
                  <li>ウェブサイトは WebAuthn を使用するため、利用者はユーザ名/パスワードの代わりにデバイスで認証できる。利用者のデバイスは、利用可能な任意のモダリティを使用できる。ノートパソコン及び形態電話での一般的な方法は、顔スキャン、指紋、PIN (個人識別番号) である。このウェブサイトは、特定の使用を強制するものではない。利用者が自分に合った方法を設定することを想定している。
```